src/types/account-id.ts

```diff
@@ -38,6 +42,10 @@ class AccountID extends Hash160 {
    * @returns an AccountID object
    */
   static fromBase58(value: string): AccountID {
+    if (!isValidClassicAddress(value)) {
```

Comment:
Unrelated to this PR?

Reply:
Yeah, I added that while I was doing the x-Address compatibility. It can either go in in this PR or the next one.

Reply:
Can we remove from this PR just to keep things minimal and in logical chunks?